website/api/controllers/webhooks/receive-from-github.js

```diff
@@ -425,7 +429,11 @@ module.exports = {
           }, baseHeaders);
         } else if (!isPRStillDependentOnAndReadyForCeoReview && existingLabels.includes('#g-ceo')) {
           // [?] https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#remove-a-label-from-an-issue
-          await sails.helpers.http.del(`https://api.github.com/repos/${owner}/${repo}/issues/${prNumber}/labels/${encodeURIComponent('#g-ceo')}`, {}, baseHeaders);
+          await sails.helpers.http.del(`https://api.github.com/repos/${owner}/${repo}/issues/${prNumber}/labels/${encodeURIComponent('#g-ceo')}`, {}, baseHeaders)
+          .tolerate('non200Response', (err)=>{
```

Comment:
```
.tolerate({ exit: 'non200Response', statusCode: 404 }, (err)=>{
```

So that we still see any unexpected errors, but only swallow the one we're expecting